alma/endpoints/payments.py

```diff
@@ -2,25 +2,22 @@
 from typing import List, Union
 
 from ..entities import Eligibility, Order, Payment
-from ..paginated_results import PaginatedResults
 from . import Base
 
 
 class Payments(Base):
     PAYMENTS_PATH = "/v1/payments"
 
     def eligibility(self, order_data):
-        response = (
-            self.request("{PAYMENTS_PATH}/eligibility".format(PAYMENTS_PATH=self.PAYMENTS_PATH))
+        return (
+            self.request(f"{self.PAYMENTS_PATH}/eligibility")
             .set_body(order_data)
             .post()
+            .expect(lambda response: Eligibility(response.json))
```

Comment:
Can't we use `expectJson(Eligibility)` here?
```suggestion
            .expectJson(Eligibility)
```